docs/src/app/components/pages/components/Dialog/ExampleCustomWidth.jsx

```diff
@@ -28,12 +28,12 @@ export default class DialogExampleCustomWidth extends React.Component {
     const actions = [
       <FlatButton
         label="Cancel"
-        secondary={true}
+        primary={true}
         onTouchTap={this.handleClose}
       />,
       <FlatButton
         label="Submit"
-        primary={true}
+        secondary={true}
```

Comment:
Yeah I agree. 

TBH With you I usually just refer to google's examples that I think look good and the material design spec itself. In the spec they keep it simple:

![image](https://cloud.githubusercontent.com/assets/4420103/13545420/d13d9ebc-e258-11e5-9405-75f32d3cda7a.png)